server/routes/github_oauth.js

```diff
@@ -79,21 +81,19 @@ router.get("/auth/github/callback", async (req, res) => {
 });
 
 async function getAccessToken(code, client_id, client_secret) {
-  const result = await axios(
-    {
+  const result = await axios({
```

Comment:
Please remove all automatic formatting like this added to this file that does not directly part of this PR.